src/openforms/authentication/contrib/digid_oidc/backends.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import SuspiciousOperation
+from django.urls import reverse
+
+from mozilla_django_oidc.auth import (
+    OIDCAuthenticationBackend as _OIDCAuthenticationBackend,
+)
+
+from openforms.authentication.constants import AuthAttribute
+
+from .mixins import SoloConfigMixin
+from .models import OpenIDConnectPublicConfig
+from .settings import OIDC_AUTHENTICATION_CALLBACK_URL
+
+logger = logging.getLogger(__name__)
+
+
+class OIDCAuthenticationDigiDBackend(SoloConfigMixin, _OIDCAuthenticationBackend):
+    """
+    Allows logging in via OIDC with DigiD
+    """
+
+    def __init__(self, *args, **kwargs):
+        self.config = OpenIDConnectPublicConfig.get_solo()
+
+        if not self.config.enabled:
+            return
```

Comment:
Do we _need_ to do this query in the `__init__`? those are expensive side-effects when instantiating python objects.

Reply:
Unfortunately I think it is, because mozilla-django-oidc tries to set the endpoints here: https://github.com/mozilla/mozilla-django-oidc/blob/master/mozilla_django_oidc/auth.py#L46. If config is not enabled, mozilla-django-oidc cannot get the settings, and raise `ImproperlyConfigured`.